<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginFormClass</class>
 <widget class="QMainWindow" name="LoginFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.988636, y1:0.995, x2:1, y2:0, stop:0 rgba(116, 172, 255, 255), stop:0.251894 rgba(99, 124, 235, 255), stop:0.598485 rgba(109, 86, 216, 255), stop:0.831439 rgba(153, 50, 204, 255));</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QWidget" name="LoginForm" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>461</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 61, 139);</string>
    </property>
    <widget class="QLabel" name="LoginTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55, 47, 107);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; color:#ffffff;&quot;&gt;Log into your ACCOUNT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LoginUsernameField">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(55, 47, 107);
border: 2px;
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter your Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LoginPasswordField">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>141</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(55, 47, 107);
border: 2px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter your Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;
background-color: rgb(48, 180, 33);
border: 1px;
border-radius: 5px;
</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginRegisterButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>460</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 50, 204);
color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;
border: 1px;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="LoginUsernameLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;

color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="LoginPasswordLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="LoginRegisterLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>420</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Don't have an account? Register now!</string>
     </property>
    </widget>
    <widget class="QLabel" name="LoginErrorLabel">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>300</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 16pt &quot;Segoe UI&quot;;
color: rgb(255, 60, 60);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="RegisterForm" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>15</y>
      <width>601</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 61, 139);</string>
    </property>
    <widget class="QLabel" name="RegisterTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55, 47, 107);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; color:#ffffff;&quot;&gt;Create a new ACCOUNT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="RegisterPasswordLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="RegisterUsernameLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;

color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RegisterUsernameField">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(55, 47, 107);
border: 2px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter your Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RegisterPasswordField">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(55, 47, 107);
border: 2px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter your Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="RegisterConfirmpasswordLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Confirm Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="RegisterEmailLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;

color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RegisterEmailField">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(55, 47, 107);
border: 2px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter your Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RegisterConfirmPassword">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(55, 47, 107);
border: 2px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm your Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RegisterButton">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>350</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(48, 180, 33);
color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;
border: 1px;
border-radius: 5px;
</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RegisterBackButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>655</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 50, 204);
color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;
border: 1px;
border-radius: 5px;
</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="RegisterErrorLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>591</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 16pt &quot;Segoe UI&quot;;
color: rgb(255, 60, 60);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="RegistrationConfirmationForm" native="true">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>210</y>
      <width>611</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 61, 139);</string>
    </property>
    <widget class="QLabel" name="RegistrationConfirmationTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55, 47, 107);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; color:#ffffff;&quot;&gt;Registration completed&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="RegistrationConfirmationText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;

color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Thank you for registering, you can now log in into your account!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="RegistrationConfirmationText_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;

color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>An information e-mail has been sent to your specified adress</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="LoggingWait" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 61, 139);</string>
    </property>
    <widget class="QLabel" name="LoadingLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 700 18pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Loading ... Please Wait</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LoginForm.qrc"/>
 </resources>
 <connections/>
</ui>
